src/main/webapp/app/overview/header-course.component.ts

```diff
@@ -49,4 +52,13 @@ export class HeaderCourseComponent implements OnChanges {
             }
         }
     }
+
+    shouldShowGoToCourseManagementButton() {
+        const courseManagementPage = this.router.url.startsWith('/course-management');
+        return !courseManagementPage && this.course.isAtLeastTutor;
+    }
+
+    redirectToCourseManagement() {
+        this.router.navigate(['course-management', this.course.id]);
+    }
```

Comment:
That's was one according to specifications by Stephan. I agree it would be nice to such buttons for other features as well. But this is a simple change, but this Par hangs there since long. How about merging this one and opening an issue for others?